zaas-service/build.gradle

```diff
@@ -89,9 +89,12 @@ dependencies {
 
     // TODO: remove awaitility from the production code, it should be just for testing purposes
     implementation libs.awaitility
+    // TODO: move to the libraries.gradle definition once finished
```

Comment:
once finishing what?

Reply:
The whole upgrade. I still hope this old library could be removed, for now it is just to be working :/